OWNERS.md

```diff
@@ -8,13 +8,29 @@ Please see the Crossplane
 [GOVERNANCE.md](https://github.com/crossplane/crossplane/blob/master/GOVERNANCE.md) for governance
 guidelines and responsibilities for the steering committee and maintainers.
 
+The Maintainers and Reviewers mirror the [crossplane/crossplane OWNERS](https://github.com/crossplane/crossplane/blob/master/OWNERS.md) with the following changes:
```

Comment:
@plumbis what is the long term intent of the docs maintainers - are we intending to always keep the two in sync? i.e. if a new maintainer is added to core crossplane, are we wanting them to also be a maintainer in this repo too?

or is this just a single point in time mirror to update docs repo one time with the core crossplane maintainers as of today?

Reply:
I agree with this sentiment - I'll approve for now. Maybe in future as we have more docs maintainers we could include only a subset of Crossplane maintainers.